src/mrfgen/mrfgen_configuration.xsd

```diff
@@ -106,8 +106,6 @@ limitations under the License.
         <xs:enumeration value="average_mp"/>
         <xs:enumeration value="average_magphase"/>
         <xs:enumeration value="mode"/>
-        <xs:enumeration value="Avg"/>
-        <xs:enumeration value="NearNb"/>
```

Comment:
Shouldn't we keep "Avg" and add "NNb" here?

Reply:
Never mind...it looks like we abstracted this to use "nearest" and "average" for users.

Reply:
right, but we should say that we support `avg` and `nnb` as values here... because we do.  I'll update this and merge